dpe/fuzz/src/fuzz_target_1.rs

```diff
@@ -82,6 +84,12 @@ fn harness(data: &[u8]) {
     };
     // There are a few vendor error codes starting at 0x1000, so this can be a 2 bytes.
     trace!("| Response Code {response_code:#06x}");
+    if dpe.contexts != prev_contexts && response_code != 0 {
+        trace!("Error: DPE state changes upon a failed DPE command.");
+    }
+    if response_code == DpeErrorCode::InternalError as u32 {
+        trace!("Error: DPE reached a state that should be unreachable.");
+    }
```

Comment:
I think we need to panic here (instead of trace) to make sure the fuzzer reports the error correctly.

Reply:
Done